saleor/webhook/payloads.py

```diff
@@ -194,6 +194,7 @@ def generate_order_payload(order: "Order"):
             "discounts": (lambda o: o.discounts.all(), discount_fields),
         },
         extra_dict_data={
+            "weight": order.weight.g if order.weight else None,
```

Comment:
Weight is already part of the `ORDER_FIELDS`. You have once pointes us an inconsistency in serializing weight for shipping methods:
```
In [19]: generate_excluded_shipping_methods_for_order_payload(o, [sm])
Out[19]: '{"order": {"type": "Order", "id": "T3JkZXI6MjE=", "channel": {"type": "Channel", "id": "Q2hhbm5lbDox", "slug": "default-channel", "currency_code": "USD"}, "shipping_method": null, "payments": null, "shipping_address": null, "billing_address": null, "discounts": null, "original": "T3JkZXI6Tm9uZQ==", "lines": [], "fulfillments": [], "private_metadata": {}, "metadata": {}, "created": "2021-10-29T10:06:30.104Z", "status": "draft", "user_email": "", "origin": "draft", "shipping_method_name": null, "shipping_price_net_amount": "0.000", "shipping_price_gross_amount": "0.000", "shipping_tax_rate": "0.0000", "total_net_amount": "0.000", "undiscounted_total_net_amount": "0.000", "total_gross_amount": "0.000", "undiscounted_total_gross_amount": "0.000", "weight": "0.0:g"}, "shipping_methods": [{"id": "U2hpcHBpbmdNZXRob2Q6MQ==", "price": "10.0", "name": "aaa", "maximum_order_weight": null, "minimum_order_weight": null, "maximum_delivery_days": 0, "minimum_delivery_days": 0}]}'
```

This is line probably going to change the format from "10:g" (`django_measurement.MeasurementField.value_to_string`) to 10.

Reply:
Thanks for catching this. I removed this as I forgot to drop it before committing 😞